src/App.js

```diff
@@ -19,12 +18,53 @@ class App extends Component {
   }
 
   componentDidMount = () => {
-    fetchAllMovies()
+    const allMovies = fetchAllMovies();
+    const faveMovies = fetchFavoriteMovies();
+
+    Promise.all([allMovies, faveMovies])
```

Comment:
very good call here, and very good implementation of a Promise.all to get this loaded in properly.